docs/software/other/mqtt.md

```diff
@@ -11,16 +11,17 @@ Meshtastic devices with wifi hardware (ESP32) are able to connect to an MQTT bro
 - Connecting your mesh to the official Meshtastic MQTT broker. This makes your devices appear on the world map, and provides a limited copy of your mesh traffic, translated into JSON.
 - Using a custom MQTT broker to bridge several mesh networks together, via the internet (or just a local IP network)
 - Using a custom MQTT broker and a translator program to decode the raw protobuf packets and translate them into a plain text form for use in other systems. eg plotting temperature readings in Grafana, or device positions in Traccar.
+- Using or emitting packets directly in/from smart home control software such as Home Assistant or other consumers that can work with JSON messages.
 
-When MQTT enabled, the Meshtastic device simply uplinks and/or downlinks every raw protobuf packet that it sees to the MQTT broker. All packets are sent to the broker, whether they originate from another device on the mesh, or the gateway node itself.
+When MQTT enabled, the Meshtastic device simply uplinks and/or downlinks every raw protobuf packet that it sees to the MQTT broker. In addition, some packet types are decoded or encoded from/to JSON messages for easier use in consumers. All packets are sent to the broker, whether they originate from another device on the mesh, or the gateway node itself.
```

Comment:
Please use Serialize/Deserialize vs encode/decode (just more accurate)

Reply:
Fixed in #6dd1de3.